daemon/mgr/spec_mount.go

```diff
@@ -73,8 +72,6 @@ func setupMounts(ctx context.Context, c *ContainerMeta, spec *SpecWrapper) error
 				}
 			}
 		}
-		s.Linux.ReadonlyPaths = nil
```

Comment:
I am wondering why to remove both two lines of code. Just curious. :)

Reply:
I move them to Linux, since they are in spec.Linux field. And only depend on if container has privileged.